test/index.html

```diff
@@ -112,5 +112,13 @@ <h3>Links: &lt;video&gt; with controls attribute can not be a child of &lt;a&gt;
         <video controls></video>
       </a>
     </section>
+    <section>
```

Comment:
Please revert all changes in this file and ensure it is up to date with `master`. 😊